docs/installingLocal.md

```diff
@@ -83,3 +83,21 @@ cd /PrairieLearn
 ./docker/test_js.sh
 ./docker/test_python.sh
 ```
+
+### Connecting to an existing Docker container
+
+The previous shells were launched in their own containers. If you want to open a shell in a Docker container that is *already running*, you can find the container's name and connect to it.
+
+1. Find the name of your running PrairieLearn container:
+
+```sh
+docker ps --format '{{.Image}}: {{.Names}}'
```

Comment:
Not so sure about this line anymore. At the time, I was worried about `docker ps` output having too much gibberish, so this would give users just a simple `prairielearn/prairielearn: adoring_gauss` output. BUT `docker ps --format '{{.Image}}: {{.Names}}'` is much harder to memorize, so it's probably worth just documenting `docker ps`.

Reply:
Agreed